woocommerce/woocommerce
# Create @woocommerce/admin-layout package

### All Submissions:






### Changes proposed in this Pull Request:




This adds a new package: `@woocommerce/admin-layout` with the following initial components moved over from `woocommerce/client/admin`:

* `WooHeaderItem`
* `WooHeaderNavigationItem`
* `WooHeaderPageTitle`
* `WooFooterItem`

More admin layout-specific components will be moved over in the future.

Closes #36721.



### How to test the changes in this Pull Request:




1. Load this branch and run `pnpm install && pnpm run build` and make sure everything builds correctly.
2. Smoke test WCAdmin pages and make sure that the headers and footers appears correctly.
   
    * Headers on all WCAdmin pages
    * CES footer on new product editor page.

3. Open your console and check if `window.wc.adminLayout` exists with the following components:

* `WC_FOOTER_SLOT_NAME`
* `WC_HEADER_NAVIGATION_SLOT_NAME`
* `WC_HEADER_PAGE_TITLE_SLOT_NAME`
* `WC_HEADER_SLOT_NAME`
* `WooFooterItem`
* `WooHeaderItem`
* `WooHeaderNavigationItem`
* `WooHeaderPageTitle`



### Other information:




### FOR PR REVIEWER ONLY:
